try.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
What does this do..?

Reply:
absolutely nothing other than embarrass me for leaving it here 😊